assets/template-collection.css

```diff
@@ -172,8 +171,9 @@
 }
 
 .collection-filters__sort {
-  font-size: 1.3rem;
+  font-size: 1.4rem;
   height: auto;
+  margin: 0;
   padding-left: 0;
   padding-right: 1.75rem;
```

Comment:
This would need `line-height: 1.5` to match the one coming from `.caption-large` for the filter summaries.

Reply:
`.collection-product-count__text` would also need it as right now it's getting a `1.4` which doesn't make it match the rest of the elements (ends up being 19px high instead of 21 like the rest). 
[Misalignment before](https://screenshot.click/21-08-myuoh-x5g00.png)
[Misalignment after you add 1.5 to `.collection-filters__sort`](https://screenshot.click/21-08-h2fbw-us5v4.png)

Reply:
Thanks for the detailed suggestion! I made the changes